bridge/cypress/support/pageobjects/UniformPage.ts

```diff
@@ -392,5 +408,15 @@ class UniformPage {
     cy.get('dt-confirmation-dialog-state').contains('Stay').click();
     return this;
   }
+
+  public assertDangerZone(): this {
+    cy.get('ktb-danger-zone').should('exist');
+    return this;
+  }
+
+  public assertDeletionDialog(): this {
+    cy.get('ktb-deletion-dialog').should('exist');
+    return this;
+  }
```

Comment:
Could you make it more generic with a status parameter?
`status ? 'exists' : 'not.exists'`

Reply:
And maybe rename it a bit? In this case, we only check for `exists` so `assertDangerZoneExists` would make it more obvious what is being checked here.